packages/promo/README.md

```diff
@@ -1,55 +1,56 @@
 # Promo
 
-The `PromoBanner` and `PromoCard` component are used to bring the user's attention to important content.
+The `PromoBanner` and `PromoCard` components are used to bring the user's attention to important content.
 
-## Types of promos
+## Types of Promos
 
 ### PromoBanner
 
 The `PromoBanner` component is used for informational content relevant to the page it's being displayed on. It typically appears below the `PageHeader`.
 
-### Customize background
+### Customize Background
 
-The background can be customized with a pre-defined set of colors or gradients. At the time of this writing, the colors and gradients don't have any semantic meaning attached to them.
+The background can be customized with a pre-defined set of colors or gradients.
 
 ### PromoInline
 
-The `PromoCard` component is less attention-grabbing than the `PromoInline`. It appears inline with the other content on the page.
+The `PromoInline` component is less attention-grabbing than the `PromoBanner`. It appears inline with the other content on the page.
 
-## Content
+## Promo Content
 
 Required content:
 
-- A headline that succintly captures the primary message of the promo
+- A headline that succinctly captures the primary message of the promo
 - Body text with more detailed information
 
 Optional content:
 
-- graphic
-- primary CTA
-- secondary CTA
+- Graphic
+- Primary CTA
+- Secondary CTA
 
 ## Hiding and showing the promo
 
 The promo will be visible until the user dismisses it.
 
 Any promo must be hidden when the user clicks the close button in the upper-right corner.
 
-Since this component is intentionally attention-grabbing, it's recommended to let users prevent the promo from re-appearing when checking the "Do not show again" checkbox.
+Since this component is intentionally attention-grabbing and can be distractive for some users, it's recommended to let users prevent the promo from re-appearing by checking the "Do not show again" checkbox.
```

Comment:
"distracting" is the word you're looking for here

Reply:
[Distractive](https://www.dictionary.com/browse/distractive) is a real word though btw.